data-raw/site-limits.R

```diff
@@ -25,6 +25,7 @@ input_site_limits <- function() {
   site_limits <- read.csv("data-raw/site-limits.csv", na.strings = c("NA", ""), stringsAsFactors = FALSE)
 
   site_limits$UpperLimit %<>% as.character()
+  site_limits$Use <- "Freshwater Life"
```

Comment:
Can we just add this column to the csv? This seems likely to cause problems if we ever change the `"data-raw/site-limits.csv"`

Reply:
yes good idea. I will add a Use column to the current site_limits.csv with all values "Freshwater Life". This will be replaced at some point with the actual data.